src/python/pants/init/specs_calculator.py

```diff
@@ -2,25 +2,17 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
The only part of this file that meaningfully changed is at the very bottom, where we convert the `--changed` result back into `Specs`.